__tests__/unit/result.test.ts

```diff
@@ -1,6 +1,7 @@
 import type { Row } from '../../src'
 import { ResultSet } from '../../src'
 import Stream, { Readable } from 'stream'
+import { guid } from '../utils'
```

Comment:
Could you remind why we generate `query_id` instead of reading from `X-ClickHouse-Query-Id` header?

Reply:
right, it will be even simpler this way